pwiz_tools/Skyline/SettingsUI/BuildLibraryDlg.Designer.cs

```diff
@@ -63,6 +63,7 @@ private void InitializeComponent()
             this.tabProperties = new System.Windows.Forms.TabPage();
             this.tabControlDataSource = new pwiz.Skyline.Controls.WizardPages();
             this.tabFilesSource = new System.Windows.Forms.TabPage();
+            this.tabAlphaSource = new System.Windows.Forms.TabPage();

```

Comment:
It doesn't seem like the form itself should need to change for this PR. Certainly no extra tab is necessary for APD